packages/jetpack-blocks/webpack.config.js

```diff
@@ -0,0 +1,155 @@
+		.filter( fs.existsSync );
+}
+/**
+ * Return a webpack config object
+ *
+ *
+ * @return {object} webpack config
+ */
+function getWebpackConfig() {
+	const workerCount = 1;
+	const cssFilename = '[name].css';
+	const preserveCssCustomProperties = true;
+
+	const presetPath = path.join( __dirname, 'src', 'preset', 'index.json' );
+	const presetIndex = require( presetPath );
+	const presetBlocks = _.get( presetIndex, [ 'production' ], [] );
+	const presetBetaBlocks = _.get( presetIndex, [ 'beta' ], [] );
+	const allPresetBlocks = [ ...presetBlocks, ...presetBetaBlocks ];
+
+	// Helps split up each block into its own folder view script
+	const viewBlocksScripts = allPresetBlocks.reduce( ( viewBlocks, block ) => {
+		const viewScriptPath = path.join( __dirname, 'src', 'blocks', block, 'view.js' );
+		if ( fs.existsSync( viewScriptPath ) ) {
+			viewBlocks[ block + '/view' ] = [ viewSetup, ...[ viewScriptPath ] ];
+		}
+		return viewBlocks;
+	}, {} );
+	//
+	// Combines all the different blocks into one editor.js script
+	const editorScript = [
+		editorSetup,
+		...blockScripts( 'editor', path.join( __dirname, 'src' ), presetBlocks ),
+	];
+	//
+	// Combines all the different blocks into one editor-beta.js script
+	const editorBetaScript = [
+		editorSetup,
+		...blockScripts( 'editor', path.join( __dirname, 'src' ), allPresetBlocks ),
+	];
+
+	const webpackConfig = {
+		bail: ! isDevelopment,
+		context: __dirname,
+		entry: {
+			editor: editorScript,
+			...( editorBetaScript && { 'editor-beta': editorBetaScript } ),
+			...viewBlocksScripts,
+		},
+		mode: isDevelopment ? 'development' : 'production',
+		devtool: process.env.SOURCEMAP || ( isDevelopment ? '#eval' : false ),
+		output: {
+			path: path.join( __dirname, 'dist' ),
+			filename: '[name].js',
+			libraryTarget: 'window',
+		},
+		optimization: {
+			minimize: shouldMinify,
+			minimizer: Minify( {
+				cache: process.env.CIRCLECI
+					? `${ process.env.HOME }/terser-cache`
+					: 'docker' !== process.env.CONTAINER,
+				parallel: workerCount,
+				sourceMap: Boolean( process.env.SOURCEMAP ),
+				terserOptions: {
+					ecma: 5,
+					safari10: true,
+					mangle: true,
+				},
+			} ),
+		},
+		module: {
+			rules: [
+				TranspileConfig.loader( {
+					workerCount,
+					configFile: path.join( __dirname, 'babel.config.js' ),
+					cacheDirectory: path.join( __dirname, '.cache' ),
+					cacheIdentifier,
+					exclude: /node_modules\//,
+				} ),
+				SassConfig.loader( {
+					preserveCssCustomProperties,
+					includePaths: [ path.join( __dirname, 'client' ) ],
+					prelude: `@import '~@automattic/calypso-color-schemes/src/shared/colors';`,
+				} ),
+				FileConfig.loader(),
+			],
+		},
+		resolve: {
+			extensions: [ '.json', '.js', '.jsx' ],
+			modules: [ 'node_modules' ],
+		},
+		node: false, // look into ?
+		plugins: _.compact( [
+			new webpack.DefinePlugin( {
+				'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
```

Comment:
We can likely remove this config dependency:
```suggestion
				'process.env.NODE_ENV': JSON.stringify( process.env.NODE_ENV ),
```

Reply:
https://github.com/Automattic/wp-calypso/pull/31503/commits/7a0909255d2d6dabf6ad4c005fb3366781f4068c